server/jetstream_cluster_3_test.go

```diff
@@ -2767,6 +2765,17 @@ func TestJetStreamClusterInterestPolicyEphemeral(t *testing.T) {
 				t.Fatalf("Expected to be able to retrieve consumer: %v", err)
 			}
 			require_True(t, info.Delivered.Stream == msgs)
+
+			// Stop the subscription and remove the interest.
+			err = sub.Unsubscribe()
+			require_NoError(t, err)
+
+			// Now wait for interest inactivity threshold to kick in.
+			time.Sleep(3 * inactiveThreshold / 2)
+
+			// Check if the consumer has been removed.
+			_, err = js.ConsumerInfo(test.stream, name)
+			require_Error(t, err)
```

Comment:
Suggest make it explicit.

require_Error(t, err, nats.ConsumerNotFound)

Reply:
ah, good idea. fixed.